tests/locator_test.go

```diff
@@ -119,6 +119,11 @@ func TestLocatorElementState(t *testing.T) {
 			`() => document.getElementById('inputText').style.visibility = 'hidden'`,
 			func(l api.Locator) bool { return l.IsVisible(nil) },
 		},
+		{
+			"hidden",
```

Comment:
I think we should have a test for when `isHidden` return `true` as well as one for when `isHidden` returns `false`.

Reply:
Hmm, but we already have a test for it:

https://github.com/grafana/xk6-browser/blob/4e0b7e659ece7fec48f36714064bfd3d48938be2/tests/locator_test.go#L138-L139